DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -923,29 +968,29 @@ public async Task RemoveMemberAsync(ulong userId, string? reason = null)
     /// Bans a specified member from this guild.
     /// </summary>
     /// <param name="member">Member to ban.</param>
-    /// <param name="delete_message_days">How many days to remove messages from.</param>
+    /// <param name="deleteMessageDays">How many days to remove messages from.</param>
     /// <param name="reason">Reason for audit logs.</param>
     /// <returns></returns>
     /// <exception cref="UnauthorizedException">Thrown when the client does not have the <see cref="DiscordPermissions.BanMembers"/> permission.</exception>
     /// <exception cref="NotFoundException">Thrown when the member does not exist.</exception>
     /// <exception cref="BadRequestException">Thrown when an invalid parameter was provided.</exception>
     /// <exception cref="ServerErrorException">Thrown when Discord is unable to process the request.</exception>
-    public async Task BanMemberAsync(DiscordUser member, int delete_message_days = 0, string? reason = null)
-        => await Discord.ApiClient.CreateGuildBanAsync(Id, member.Id, delete_message_days, reason);
+    public async Task BanMemberAsync(DiscordUser member, int deleteMessageDays = 0, string? reason = null)
+        => await Discord.ApiClient.CreateGuildBanAsync(Id, member.Id, deleteMessageDays, reason);
 
     /// <summary>
     /// Bans a specified user by ID. This doesn't require the user to be in this guild.
     /// </summary>
-    /// <param name="user_id">ID of the user to ban.</param>
-    /// <param name="delete_message_days">How many days to remove messages from.</param>
+    /// <param name="userId">ID of the user to ban.</param>
+    /// <param name="deleteMessageDays">How many days to remove messages from.</param>
     /// <param name="reason">Reason for audit logs.</param>
     /// <returns></returns>
     /// <exception cref="UnauthorizedException">Thrown when the client does not have the <see cref="DiscordPermissions.BanMembers"/> permission.</exception>
     /// <exception cref="NotFoundException">Thrown when the member does not exist.</exception>
     /// <exception cref="BadRequestException">Thrown when an invalid parameter was provided.</exception>
     /// <exception cref="ServerErrorException">Thrown when Discord is unable to process the request.</exception>
-    public async Task BanMemberAsync(ulong user_id, int delete_message_days = 0, string reason = null)
-        => await Discord.ApiClient.CreateGuildBanAsync(Id, user_id, delete_message_days, reason);
+    public async Task BanMemberAsync(ulong userId, int deleteMessageDays = 0, string? reason = null)
+        => await Discord.ApiClient.CreateGuildBanAsync(Id, userId, deleteMessageDays, reason);

```

Comment:
should be changed to deleteMessageSeconds, and we should put up a breaking change warning

Reply:
Should we just change it to a timespan and pass `TotalSeconds` to the API?

Reply:
Yeah, im on it

Reply:
considered doing that for v6, we can totally do that already